react/ActionMenu/index.jsx

```diff
@@ -101,6 +103,26 @@ const ActionMenu = ({
   anchorElRef,
   containerElRef
 }) => {
+  const theme = useTheme()
+  const sidebar = document.querySelector('[class*="sidebar"]')
+
+  useSetFlagshipUI(
```

Comment:
nit: add a test checking useSetFlagshipUI is called with correct arguments, when this component is rendered